ingestion/src/metadata/ingestion/source/kafka.py

```diff
@@ -108,7 +118,8 @@ def next_record(self) -> Iterable[CreateTopicRequest]:
                     logger.info("Fetching topic schema {}".format(topic_name))
                     topic_schema = self._parse_topic_metadata(topic_name)
                     logger.info("Fetching topic config {}".format(topic_name))
-                    topic_request = CreateTopicRequest(
+                    topic = Topic(
```

Comment:
Hi @harshach, this change looks like a regression to me. We should be aiming at using `Create` requests for the connectors

Reply:
Let's tackle this as part of https://github.com/open-metadata/OpenMetadata/issues/1673